vit-servicing-station-tests/src/tests/mod.rs

```diff
@@ -1,3 +1,5 @@
 pub mod bootstrap;
 pub mod graphql;
+#[cfg(feature = "non-functional")]
+pub mod non_functional;
```

Comment:
Why do we want to activate/deactivate this testing? Is this gonna run in some other runners?

Reply:
usually non_functional tests are a bit longer (up to 24 hours). so we probably don't want to run them during CI

Reply:
Got it. Thanks!